it/ApiConnectorISpec.scala

```diff
@@ -0,0 +1,39 @@
+import cats.effect.unsafe.implicits.global
+import cats.implicits._
+
+import uk.gov.nationalarchives.omega.editorial.connectors.ApiConnector
+import uk.gov.nationalarchives.omega.editorial.editSets
+
+class ApiConnectorISpec extends BaseISpec {
+
+  private val idOfExistingEditSet = "1"
+
+  lazy val apiConnector: ApiConnector = app.injector.instanceOf[ApiConnector]
+
+  "when a request for an edit set is made" must {
+    s"get an edit set for id: $idOfExistingEditSet" in {
+      val editSetResponse = apiConnector.getEditSet(idOfExistingEditSet).unsafeRunSync()
+
+      editSetResponse mustBe editSets.editSet1
+    }
+
+    s"get the edit set twice for id: $idOfExistingEditSet" in {
```

Comment:
I wonder if we need this.

In order to verify the request / reply client? 

If so, we already do that.

Also, as we're returning the same edit set, there's no way to catch out an error by which one request gets the response meant for the other - which I think is the intent of such a test: to confirm the pairing.

Reply:
I just thought that it would demonstrate that once the request-reply client can handle 2 messages in succession, i.e. one is pulled off the queue before the other.

But I think you are right, that is a test for the request-reply client spec. I think I'll delete this one.

Reply:
Confirmed.